boltstub/wiring.py

```diff
@@ -251,10 +352,50 @@ def remote_address(self):
         """
         return Address(self.__socket.getpeername())
 
+    @property
+    def _socket(self):
+        return self.__socket
+
```

Comment:
This looks odd. If a property is marked as internal (i.e. starts with `_`) why even bother exposing it?

Reply:
I exposed it because during the tests I need to check if the Wire was created with the correct socket type.